include/C#/SAMP_API.cs

```diff
@@ -121,19 +143,44 @@ class API
         [DllImport(PATH, CallingConvention = CallingConvention.Cdecl)]
         public static extern float GetVehicleHealth();
         [DllImport(PATH, CallingConvention = CallingConvention.Cdecl)]
-        public static extern short GetVehicleTypeId();
+        public static extern int GetVehicleModelId();
         [DllImport(PATH, CallingConvention = CallingConvention.Cdecl)]
-        public static extern int IsVehicleBike();
-
+        public static extern int GetVehicleModelName(ref StringBuilder name, int len);
+        public static extern int GetVehicleModelNameById(int vehicleId, ref StringBuilder name, int len);
         [DllImport(PATH, CallingConvention = CallingConvention.Cdecl)]
-        public static extern int GetPlayerIDByName(string name);
+        public static extern int GetVehicleType();
         [DllImport(PATH, CallingConvention = CallingConvention.Cdecl)]
-        public static extern int GetPlayerNameByID(int id, ref StringBuilder playerName, int maxLen);
-
+        public static extern int GetVehicleFreeSeats(ref int seatFL, ref int seatFR, ref int seatRL, ref int seatRR);
         [DllImport(PATH, CallingConvention = CallingConvention.Cdecl)]
-        public static extern int Init();
+        public static extern int GetVehicleFirstColor();
         [DllImport(PATH, CallingConvention = CallingConvention.Cdecl)]
-        public static extern void SetParam(string _szParamName, string _szParamValue);
+        public static extern int GetVehicleSecondColor();
+        [DllImport(PATH, CallingConvention = CallingConvention.Cdecl)]
+        public static extern int GetVehicleColor(ref int color1, ref int color2);
```

Comment:
Mh...since we are always binding a value to the parameters ([#L118](https://github.com/SAMPProjects/Open-SAMP-API/blob/c68dffbaa07b472e4191a6acf97b55c52b63b6db/src/Open-SAMP-API/Client/VehicleFunctions.cpp#L118)), which is required for `out`, we should be good with the change. (The initial value is also not needed, so yea, it would make sense to change it, good catch!)